libs/ledger-live-common/src/families/hedera/api/network.ts

```diff
@@ -37,9 +38,15 @@ export async function getAccountBalance(
   address: string
 ): Promise<AccountBalance> {
   const accountId = AccountId.fromString(address);
-  const accountBalance = await new hedera.AccountBalanceQuery({
-    accountId,
-  }).execute(getClient());
+  let accountBalance;
+
+  try {
+    accountBalance = await new hedera.AccountBalanceQuery({
+      accountId,
+    }).execute(getClient());
+  } catch {
+    throw new HederaAddAccountError()
```

Comment:
Missing semicolon here, prettier is complaining :slightly_smiling_face: 
https://github.com/LedgerHQ/ledger-live/actions/runs/3137430722/jobs/5095587144#step:10:2848

Protip: you can setup your IDE to automatically apply prettier formatting every time you save a file.

Reply:
Thanks for the tip! I should really put that in habit haha